OgraTests/OgraModel.swift

```diff
@@ -67,3 +102,10 @@ extension Pet: Decodable, Encodable {
 		])
 	}
 }
+
+extension Continent: Decodable, Encodable {}
+
+extension IntBulls: Decodable, Encodable {}
+extension DoubleBulls: Encodable {}
+extension FloatBulls: Encodable {}
+extension UIntBulls: Encodable {}
```

Comment:
Argo doesn't provide a default implementation for `Double`, `Float`, or `UInt`.

I added them here because those types were supported for `.Number` but without support on both sides I'm not sure how useful they are.